pymc/distributions/multivariate_utils.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+
+from aesara.graph.basic import Apply
+from aesara.graph.op import Op
+from aesara.tensor import swapaxes
+from aesara.tensor.basic import as_tensor_variable
+
+
+class BatchedMatrixInverse(Op):
+    """Computes the inverse of a matrix.
+
+    `aesara.tensor.nlinalg.matrix_inverse` can only inverse square matrices.
+    This Op can inverse batches of square matrices.
+    """
```

Comment:
Two questions:
- Is there an existing function in aesara that can inverse batches of square matrices in a vectorized way?
- If no, I have added this `Op` in a new file. Where should this class ideally be placed?

Reply:
`distributions/dist_math.py`?

Reply:
If numpy has support for batched matrix inversion but not Aesara, we can open an there. Here is a similar case: https://github.com/aesara-devs/aesara/issues/791

Reply:
Actually the Aesara Op is just doing the same as this one but has an unnecessary ndim check? 

We should just open a PR to fix it there